### ISSUE TITLE
--git-tag does not commit `Cargo.lock`

### ISSUE THREAD

**Issue by zkat** [wraithan/cargo-bump#24]
Every time I do `cargo bump -g` now, I get `Cargo.lock` as a modified file in my project root, after the commit :(

**Comment by wraithan** [comment]
I thought it was the next `cargo build` after `cargo bump -g` which then updates the `Cargo.lock`. Not the `cargo bump -g` itself. I can test that soon to be sure.

**Comment by rubik83** [comment]
This bug is also visible in the release of cargo-bumb itself
In this zip : 
https://github.com/wraithan/cargo-bump/archive/1.1.0.zip
In Cargo.lock line 65 : version = "1.0.2"
The version of cargo-bump is bad

**Comment by gibfahn** [comment]
>I thought it was the next cargo build after cargo bump -g which then updates the Cargo.lock. Not the cargo bump -g itself. I can test that soon to be sure.

Yeah this is true, but ideally the bump commit would include the change to the Cargo.lock as well.

**Comment by Stargateur** [comment]
`cargo bump -g` should do a `cargo check`

**Comment by LevitatingBusinessMan** [comment]
I solved this in my branch by having cargo generate the lockfile
